corehq/apps/fixtures/tasks.py

```diff
@@ -40,3 +42,19 @@ def async_fixture_download(table_ids, domain, download_id):
     DownloadBase.set_progress(task, 0, 100)
     prepare_fixture_download(table_ids, domain, task, download_id)
     DownloadBase.set_progress(task, 100, 100)
+
+
+@task(queue='background_queue', bind=True, default_retry_delay=15 * 60)
+def delete_unneeded_fixture_data_item(self, domain, data_type_id):
```

Comment:
Does this task need to clear any caches?

Reply:
https://github.com/dimagi/commcare-hq/pull/24292/commits/fb5e1d0af145df429ce78a1849186576491ec905

I chose to only document this because I think this might open a whole can of worms that I'd need to investigate in current lookup tables. 

Because there is no fixture ownership supported by the upload that uses this, there should be no issues with not busting a cache. That task (_run_fast_fixture_upload) handles deleting the data type and busting the cache that is for returning data_items by data type. All global fixtures are only accessed via getting the items by data type id.

It appears that non-global fixture data items can be referenced only by FixtureOwnership (side stepping data_type_id). That's not ideal, but its not currently being used in a way that would cause issues.

I think this also means that there's potential for current projects to have data items that do not have a data type, depending on how lookup table uploads have failed

Reply:
sounds reasonable